provisioning/transport/amqp/devdoc/amqp_requirements.md

```diff
@@ -34,6 +34,8 @@ and shall set the password to the passed in sas token.
 
 **SRS_NODE_PROVISIONING_AMQP_16_002: [** The `registrationRequest` method shall connect the AMQP client with the certificate and key given in the `auth` parameter of the previously called `setAuthentication` method. **]**
 
+**SRS_NODE_PROVISIONING_AMQP_99_026: [** The `registrationRequest` method shall connect the AMQP client with the SSL agent given in the `auth` parameter of the previously called `setAuthentication` method. **]**
```

Comment:
nit: shouldn't this be AMQP_99_001

Reply:
My bad, misunderstood the following:
> use 99 for a developer id, and just increment the last number of the requirement to be unique
I assumed the second number sequence has to be unique.

Reply:
Fixed.